langchain/src/document_loaders/web/browserbase.ts

```diff
@@ -0,0 +1,83 @@
+import { Document } from "@langchain/core/documents";
+import type { BrowserbaseLoadOptions } from "@browserbasehq/sdk";
+import { BaseDocumentLoader } from "../base.js";
+import type { DocumentLoader } from "../base.js";
+
+type BrowserbaseLoaderOptions = BrowserbaseLoadOptions & {
+  apiKey?: string;
```

Comment:
Also, not sure if you're aware, but we have a [`getEnvironmentVariable`](https://api.js.langchain.com/functions/langchain_core_utils_env.getEnvironmentVariable.html) util func you can use to optionally load env vars.

Reply:
apiKey can be undefined, we deal with it in the underlying library